integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-fuse-conf.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "alluxio.fullname" . }}-fuse-config
+  annotations:
+    "helm.sh/hook": "pre-install"
+    "helm.sh/hook-delete-policy": before-hook-creation
+  labels:
+    name: {{ template "alluxio.fullname" . }}-fuse-config
+    app: {{ template "alluxio.name" . }}
+    chart: {{ template "alluxio.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  ALLUXIO_JAVA_OPTS: |-
```

Comment:
why do we need a separate configMap for fuse? we should be able to use `ALLUXIO_FUSE_JAVA_OPTS` for any fuse specific java opts (https://github.com/Alluxio/alluxio/blob/master/integration/fuse/bin/alluxio-fuse#L48)